src/components/Filters/DimensionFilter.tsx

```diff
@@ -167,7 +174,7 @@ const DimensionFilter: FC<DimensionFilterProps> = ({
           </p>
         )}
       </div>
-      {filter}
+      <div aria-labelledby={filterLabelID}>{filter}</div>
```

Comment:
Ah I thought you were talking about using the `aria-labelledby` on the button to associate "Get Access" with the button, which is why I mentioned that it in the accessible name it would override the contents of the button. 

What I was suggesting was adding `aria-describedby` pointing to the Get Access id so it might sound something like this:

<img width="888" alt="Screenshot 2024-10-22 at 1 27 40 AM" src="https://github.com/user-attachments/assets/04654d60-3f52-4534-8b0c-8c401a5ec31b">